tests/test_cli.py

```diff
@@ -82,3 +82,27 @@ def test_list(artifact, n_files, capsys):
 
     with pytest.raises(ValueError):
         cli.main(["list", "setup.py"])
+
+
+@pytest.mark.parametrize(
```

Comment:
Several other conda projects including conda, conda-index, conda-package-streaming, use local web servers as a CI fixture for this sort of thing.

Reply:
Added a bottle fixture similar to what CPS is doing (but supported by pytest-xprocess for simplicity).